docs/guide/topics/threads.rst

```diff
@@ -0,0 +1,245 @@
+
+We touched on it earlier a bit, but now we'll go into how to add people to threads.
+
+The go-to practice for doing this is pinging the member, like so:
+
+.. image:: /images/guide/threads/mention_member_to_thread.png
+
+The API has methods of adding members that do not require pinging them, which is implemented with :meth:`Thread.add_user`.
+You can add it as a command, like so:
+
+.. code-block:: python3
+    :emphasize-lines: 10
+
+    async def is_thread():
+        def predicate(ctx: commands.Context) -> bool:
+            return isinstance(ctx.channel, discord.Thread)
+        return commands.check(predicate)
+
+    @bot.command()
+    @is_thread() # we use the defined check above so that this command can only be used in a thread.
+    async def add_to_thread(ctx: commands.Context, *, member: discord.Member) -> None:
+        assert isinstance(ctx.channel, discord.Thread) # guarded by decorated check!
+        await ctx.channel.add_user(member)
+        await ctx.message.add_reaction("\U00002705")
+
+Which performs the following:
+
+.. image:: /images/guide/threads/add_member_to_thread.png
+
+Another method in which to do so is to mention a role within the thread.
+
+.. note::
+    There is a limit of around 100 members per addition to a thread, so if your role contains more than 100 members they will not be added.
+
+
+Setting the archive duration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Threads automatically archive after a set period of time of inactivity. You can choose this period when creating the thread.
+The current accepted values are:
+
+    - 1 hour
+    - 24 hours
+    - 3 days
+    - 1 week
+
+
+.. note::
+    The options for both "3 days" and "1 week" are locked behind server boost level 1 and 2, respectively.
+
+
+To pass an auto-archive duration during thread creation, you can use the ``auto_archive_duration`` keyword argument to the :meth:`TextChannel.create_thread` call:
+
+.. code-block:: python3
+    :emphasize-lines: 5
+
+    async def create_daily_thread(ctx: commands.Context):
+        duration = 1440 # 24 hours represented as minutes
+        await ctx.channel.create_thread(
+            name="Daily thread!",
+            auto_archive_duration=duration,
+            message=ctx.message
+        )
+
+Deleting or archiving a thread
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can delete or force archive threads if you have the :attr:`~Permissions.manage_threads` permission.
+
+If you wish to force archive the thread you can use the :meth:`Thread.edit` method and set the ``archived=True`` keyword argument.
+
+To delete a thread you can use the :meth:`Thread.delete` method.
+
+
+Browsing threads in a channel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can browse threads within :class:`~TextChannel` instances with the following methods:
+    - :attr:`~TextChannel.threads`
+    - :meth:`~TextChannel.archived_threads`
+
+The former is a property of the channel instance that returns a list of all valid :class:`~Thread` instances
```

Comment:
What does "valid" mean here?

Also should be important to mention in this section that `TextChannel.threads` will only contain active threads (excluding all archived threads which can only be fetched with the `archived_threads` iterator or `fetch_thread()`. Not even `channel.get_thread()` will get them since archived threads are not cached, they're evicted from the cache once they are archived even.)

Reply:
The lines after your comment here do cover the rest of this comment.
I have fixed the "valid" mention locally, thank you